wasmcloud-wasi-kv/wasmcloud-wasi-kv-wit/tests/keyvalue.rs

```diff
@@ -0,0 +1,52 @@
+
+/// Build a fused component for 
+async fn build_fused_component {
+    
+}
+
+#[tokio::test]
+async fn actor_http_log_rng_component() -> anyhow::Result<()> {
```

Comment:
:thinking:

Reply:
oh woops, yeah this is actually something I was going to use to fill out -- turns out `wasmtime` doesn't build quite so easily here, I had to go back to using your repo

- [ ] remove unfinished tests for keyvalue stuff

Reply:
oh woops, yeah this is actually something I was going to use to fill out -- turns out `wasmtime` doesn't build quite so easily here, I had to go back to using your repo

- [ ] remove unfinished tests for keyvalue stuff